roles/setup_pgd/defaults/main.yml

```diff
@@ -113,3 +113,27 @@ supported_pg_version:
 
 supported_pgd_version:
   - 5
+
+# The following parameters must be configured in the 'pgd_commit_scopes' list
+# the values below are examples and should be adjusted to your requirements
+#  for CAMO
+#  - cs_type: 'CAMO'
+#  - cs_name: 'camo_scope_1'
+#  - parent_group: 'pgd_cluster'
+#  - cs_origin_node_group: 'pgd_two_nodes'
+#  - member_nodes: ['edb-primary1', 'edb-primary2']
+#  - default_group_cs: true
+#  - cs_rule: "ALL ( pgd_two_nodes ) ON visible CAMO DEGRADE ON (timeout=500s) TO ASYNC"
+#  for GROUP_COMMIT
+#  - cs_type: 'GROUP_COMMIT'
+#  - cs_name: 'group_commit_scope_1'
+#  - parent_group: 'pgd_cluster'
+#  - cs_origin_node_group: 'pgd_gc_nodes'
+#  - member_nodes: ['edb-primary3']
+#  - default_group_cs: true
+#  - cs_rule: "ALL ( pgd_remaining_nodes ) GROUP COMMIT"
+
+# For CAMO Commit Scopes
+max_prepared_transactions: 100
+
+pgd_commit_scopes: ""
```

Comment:
All of this should be moved in front of the `supported_pgd_version` var.

Reply:
Will move